primer/brand
# CTAForm Component

## Summary

Creating a CTAForm component that most commonly will be used to create subscribe CTA blocks for marketing pages. It can be applied for other use cases.

## Deployments:
[MDX Documentation](https://primer-7a98f9a2bb-26139705.drafts.github.io/components/CTAForm)
[Storybook](https://primer-7a98f9a2bb-26139705.drafts.github.io/storybook/?path=/docs/components-ctaform--playground)

## List of notable changes:

- **added** CTAForm component

## What should reviewers focus on?

- Making sure the component functions and looks like it is supposed to.

## Steps to test:

1. Open the CTAForm component in CI-deployed preview environment
2. Go to CTAForm story in Storybook
3. Verify that the CTAForm behaves as described in the following issue.

## Supporting resources (related issues, external links, etc):

- [Figma Design](https://www.figma.com/file/BJ95AjraesmRCWsKA013GS/branch/LSoPddDUuzfy4RLRh4JyPv/Primer-Brand?node-id=4482%3A29680&t=rolx6DbuNYsjYd8h-0)

## Contributor checklist:


## Reviewer checklist:


## Screenshots:

![image](https://user-images.githubusercontent.com/31220082/213505696-0f895b04-6468-428c-9da2-eaf57e6b6c68.png)
